account_invoice_date_due/models/account_move.py

```diff
@@ -14,8 +14,9 @@ class AccountMove(models.Model):
     @api.onchange("invoice_date_due_payment_term")
     def _onchange_invoice_date_due_payment_term(self):
         """Propagate from Payment term due date to original field"""
-        if self.invoice_date_due_payment_term:
-            self.invoice_date_due = self.invoice_date_due_payment_term
+        for move in self:
```

Comment:
onchange is by design single-record, so this change is not needed.

Reply:
@pedrobaeza it is only explicitly api.onchange. it is also implicitly api.multi and callable outside of an onchange context.

Reply:
All methods are by default `api.multi`, but as convention, all the onchanges are considered single-record. See all of them in across Odoo core. Anyways, this is not blocking.

Reply:
I agree iterating is not required, it could give the wrong impression it's actually necessary and adds an extra indentation level